mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
okay, so how about this:

```py
# specify train and test window only
ep_slice = EpochsSlicer(epochs=epochs, train_time=(0.2, 0.3), test_time=(0.5, 0.6))
svm = LinearSVM()
clf = Pipeline(('ep_slice', ep_slice), ('svm', svm))
clf.fit_transform(epochs_array)
```
Basically, it is one cell in the GAT array that will be the `TimeSlicer` estimator and the `GAT` object gets refactored into convenience plotting functions.